### DISCUSSION TITLE
请问：多线程并发查询数据时，偶尔会出现未响应问题，日志里也出现任何错误信息，请问该如何分析排查

### DISCUSSION THREAD

**Discussion by cixiancong01** [taosdata/TDengine#23820] [Q&A]
现象为：程序多线程并发查询数据时，偶尔会出现无响应问题，现象像是库连接锁死，目标表再开个客户端去访问也是无响应了。有时候通过关闭查询客户端，能够恢复查询功能。有时候关闭了也没有反应，查询数据库状态变为0，此时只有重启数据库服务才能恢复。刚开始接触了解TD数据库，想请问下该如何分析该问题，谢谢!

**Comment by paven627** [comment]
是有这个问题，遇到过多次， 查询频率越高触发几率越大， 是jdbc驱动的bug，最后没办法，更新数据库和驱动好像好了

**Reply by cixiancong01** [reply]
哦，谢谢！你用的是哪个版本的jdbc驱动，我去试试
